dissect/target/plugins/os/unix/services.py

```diff
@@ -112,37 +113,19 @@ def parse_systemd_config(fh: BinaryIO) -> str:
     This should probably be rewritten to return a proper dict as in
     its current form this is only useful when used in Splunk.
     """
-    variables = ""
+    parser = RawConfigParser(strict=False)
```

Comment:
The default delimiters are `=` and `:`, while systemd only allows `=`.

Also multi line values seem to must have a `\` at the end of the line (and possibly indenting of the next line), where the default for `RawConfigParser` is just indenting of the next line. How does it handle this?

systemd config allows empty values but with a delimiter (`=`), which is different from the (disabled by default) option `allow_no_value` which allows for keys without a delimiter all together. Are these empty values handled properly?